blocknative/utils.py

```diff
@@ -34,31 +34,25 @@ def status_error_to_exception(message: dict) -> None:
         SDKError: If there is a non-specific error that is indicated by the server.
     """
 
-    if message["status"] == "ok" or "reason" in message:
+    if message['status'] == "ok" or not "reason" in message:
```

Comment:
@taylorjdawson Please double check the "no" error logic is as it should be here - "not "reason"

Reply:
Reason will be there for non error messages as well so maybe it should be:
```python
if message["status"] == "ok" or not message["reason"]:
```

Reply:
Updated.